indico/htdocs/sass/base/_animation.scss

```diff
@@ -18,6 +18,7 @@
 /* Animation utility constructs */
 /* To be removed whenever we support a more recent version of Compass */
 
+@import 'base/palette';
```

Comment:
Maybe we should rather import palette before everything else instead of using strictly alphabetical order in `_base.scss`?